src/elementPlugins/dropdownChoice/src/TemplateView.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { useState } from 'react'
+import { Input } from 'semantic-ui-react'
+import { TemplateViewProps } from '../../types'
+
+// TODO type should be global
```

Comment:
Is this still todo? What type should be global?

Reply:
I don't know  -- this is left over from Andrei's original prototype, I haven't changed this at all, it's just copied from shortText folder. AFAIK, we're not really dealing with the TemplateViews at the moment, as that will be for the template builder.